pandas/core/tools/numeric.py

```diff
@@ -118,10 +123,14 @@ def to_numeric(arg, errors="raise", downcast=None):
     is_series = False
     is_index = False
     is_scalars = False
+    is_numeric_extension_dtype = False
 
     if isinstance(arg, ABCSeries):
         is_series = True
         values = arg.values
+        if is_extension_array_dtype(arg) and isinstance(values, NumericArray):
+            is_numeric_extension_dtype = True
+            values = extract_array(arg)
```

Comment:
Is this extract_array line needed? (the `values = arg.values` from above should have worked fine, I think)

Reply:
It's not, I reverted this bit